simtools/layout/layout_array.py

```diff
@@ -195,6 +223,13 @@ def _from_corsika_file_to_dict(self, file_name=None):
                     )
                     corsika_dict[simtools_par][tel_type] = corsika_dict[simtools_par][tel_type]
 
+        if self.mongo_db_config is None:
+            self._logger.error("DB connection info was not provided, cannot set telescope altitude")
+            raise ValueError
+        if self.site is None:
+            self._logger.error("Site was not provided, cannot set telescope altitude")
+            raise ValueError
```

Comment:
Just to make sure we are on the same line of thoughts. Once we implement the feature to get the site from `telescope_list_file` in case the file is passed as argument (as discussed in #436), we will also accept `self.site` and `self.mongo_db_config` as None and will not raise these errors. is that right?

Reply:
As I mentioned above, we already accept them as `None` if we provide a `telescope_list_file`. What will change later is that we could read the site from the file sometimes. I suggest we have a quick chat before we start implementing #436 to make sure it's all clear.